README.md

```diff
@@ -16,7 +16,7 @@ At UPSIDER, we have a few internal repositories set up with a monorepo structure
 
 We are looking to add a few more features, such as extra signoff from non-coder, label based check, etc.
 
-<sup><sub>NOTE:  
```

Comment:
If you take out the blank spaces here, it would change the line break layout slightly. While it seems to read just fine with this change, I'd like to check if this was an intentional update?

Reply:
my ide removed it